iFixit/checkout-ui-extensions
# Setup Sentry

Closes https://github.com/iFixit/checkout-ui-extensions/issues/24

Loosely follows Shopify's suggestions [here](https://shopify.dev/docs/api/checkout-ui-extensions/2024-01/error-handling#sentry).

## QA

- Uncaught errors in local dev should be reported to the [checkout-dev](https://ifixit.sentry.io/projects/checkout-dev) project in Sentry.
- The links on the thank you page should still work properly (see QA from https://github.com/iFixit/checkout-ui-extensions/pull/49).
